index.html

```diff
@@ -101,6 +101,11 @@ <h1>ESP Web Tools</h1>
           your ESP device, you might miss <a href="#drivers">drivers</a>.
         </i>
       </p>
+      <p>
```

Comment:
We should only make this visible when we start flashing. The button emits events that we can hook into.